src/index.tsx

```diff
@@ -139,44 +144,48 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
             if (copyStyles !== false) {
                 const headEls = document.querySelectorAll("style, link[rel='stylesheet']");
 
-                for (let index = 0, l = headEls.length; index < l; ++index) {
-                    const node = headEls[index];
+                for (let i = 0, headElsLen = headEls.length; i < headElsLen; ++i) {
+                    const node = headEls[i];
                     if (node.tagName === "STYLE") {
                         const newHeadEl = domDoc.createElement(node.tagName);
                         const sheet = (node as HTMLStyleElement).sheet as CSSStyleSheet;
 
                         if (sheet) {
                             let styleCSS = "";
-                            for (let i = 0; i < sheet.cssRules.length; i++) {
-                                if (typeof sheet.cssRules[i].cssText === "string") {
-                                    styleCSS += `${sheet.cssRules[i].cssText}\r\n`;
+                            // NOTE: for-of is not supported by IE
+                            for (let j = 0, cssLen = sheet.cssRules.length; j < cssLen; j++) {
```

Comment:
The outer loop increments with `++i` while the inner loop increments by `j++` (also uses `++j` below). Can we please make this consistent? Could see this leading to a bug down the line.